app/components/access-level-pill.hbs

```diff
@@ -1,62 +1,50 @@
-<div class="auk-o-flex auk-o-flex--vertical-center">
-  {{#if this.isLoading}}
-    <div class="auk-loader" role="alert" aria-busy="true"></div>
-  {{else}}
-    {{#if (not @confidential)}}
-      {{#if (not this.isEditing)}}
-        <Auk::Pill
-          @skin={{this.pillSkin}}
-          class="auk-u-cursor-pointer"
-          data-test-access-level-pill
-          {{on "click" this.toggleEdit}}
-        >
-          {{this.accessLevelLabel}}
-        </Auk::Pill>
-      {{else}}
-        <div class="auk-u-width-3/4">
-          <Utils::ModelSelector
-            data-test-access-level-pill-selector
-            @modelName="access-level"
-            @allowClear={{true}}
-            @propertyToShow="label"
-            @sortField="priority"
-            @selectedItems={{@accessLevel}}
-            @selectModel={{this.changeAccessLevel}} />
-        </div>
-        <Auk::Button
-          @skin="borderless"
-          @layout="icon-only"
-          @icon="check"
-          {{on "click" (perform this.confirmChangeAccessLevel @accessLevel)}}
-          data-test-access-level-pill-save
-        >
-          {{t "more-options"}}
-        </Auk::Button>
-        <Auk::Button
-          @skin="borderless"
-          @layout="icon-only"
-          @icon="x"
-          {{on "click" (perform this.cancelChangeAccessLevel)}}
-          data-test-access-level-pill-cancel
-        />
-      {{/if}}
+{{#if this.isLoading}}
+  <div class="auk-loader" role="alert" aria-busy="true"></div>
+{{else}}
+  {{#if (not this.isEditing)}}
+    <Auk::Pill
+      @skin="active"
+      @icon={{this.pillIcon}}
+      data-test-access-level-pill
+    >
+      {{this.accessLevelLabel}}
+    </Auk::Pill>
+    {{#if this.canEdit}}
+      <Auk::Button
+        @skin="tertiary"
+        @layout="icon-only"
+        @icon="pencil"
+        @size="tiny"
+        {{on "click" this.openEditMode}}
+        data-test-access-level-pill-edit
+      />
     {{/if}}
+  {{else}}
+    <div class="auk-u-maximize-width">
+      <Utils::ModelSelector
```

Comment:
@sergiofenoll Ik zag nu zonet binnen de `usertest-archive` branch nog wat bijkomende (nieuwe) styling gekoppeld aan dit component (ik ben nu namelijk bezig met https://kanselarij.atlassian.net/browse/KAS-3410). Binnen jous aanpassingen is dit niet mee gekopieerd. Vermoedelijk was/is de styling daar rond in orde zonder die extras binnen https://github.com/kanselarij-vlaanderen/frontend-kaleidos/blob/usertest-archive/app/styles/prototype/_agenda.scss?

<img width="396" alt="CleanShot 2022-05-17 at 23 51 27@2x" src="https://user-images.githubusercontent.com/18174827/168916132-121365d3-b3cd-4e75-b366-9d342d289cb8.png">

Reply:
Merci voor de heads up! Heb die `.auk-model-selector` klasse & CSS toegevoegd, maar ik zag niet direct een verschil met en zonder, dus ik heb het niet gecommit.